render/detailed/links.go

```diff
@@ -0,0 +1,169 @@
+package detailed
+
+import (
+	"bytes"
+	"net/url"
+	"strings"
+	"text/template"
+
+	"github.com/weaveworks/scope/probe/docker"
+	"github.com/weaveworks/scope/report"
+
+	"github.com/ugorji/go/codec"
+)
+
+// MetricLink describes a link referencing a metric.
+type MetricLink struct {
+	// References the metric id
+	ID       string `json:"id,omitempty"`
+	Label    string `json:"label"`
+	URL      string `json:"url"`
+	Priority int    `json:"priority"`
+}
+
+// Variable name for the query within the metrics graph url
+const urlQueryVarName = ":query"
+
+var (
+	// As configured by the user
+	metricsGraphURL = ""
```

Comment:
I talked about this with @2opremio; I first went the route of _passing it down_ which lead to exponential increase of function signature adjustments (I think I stopped when I crossed 10 and saw there is no end in sight).

I'll take another look at it and am happy to receive alternative suggestions in the meantime.